src/com/yetanalytics/lrs_admin_ui/views/status.cljs

```diff
@@ -0,0 +1,227 @@
+              "")}
+    (or value " ")]
+   [:div.big-number-label
+    label]]))
+
+(defn timestamp
+  [vis-key label sub-qvec]
+  (let [value @(subscribe sub-qvec)]
+    [:div.timestamp
+     [:div.timestamp-label
+      label]
+     [:div.timestamp-value
+      {:class (if @(subscribe [:status/loading? vis-key])
+                "spinner"
+                "")}
+      (or value " ")]]))
+
+(defn refresh-button
+  []
+  [:div.status-refresh-button
+   [:input.btn-blue-bold
+    {:type "button"
+     :value "REFRESH"
+     :on-click #(dispatch [:status/get-all-data])}]])
+
+(defn title-loading-spinner
+  [vis-key]
+  [:span.vis-title-loading-spinner
+   {:class (if @(subscribe [:status/loading? vis-key])
+             "spinner"
+             "")}])
+
+(defn platform-pie
+  []
+  [:div.vis-pie
+   [:h4 "PLATFORMS"
+    [title-loading-spinner "platform-frequency"]]
+   [vis/pie
+    {:theme (aget vis/theme "material")
+     :data @(subscribe [:status.data/platform-frequency])
+     :labels (fn [c]
+               (let [datum (aget c "datum")
+                     x (aget datum "x")
+                     y (aget datum "y")]
+                 (format "%s: %s"
+                         x y)))}]])
+
+(defn timeline-select-unit
+  []
+  (let [current-unit @(subscribe [:status.params/timeline-unit])]
+    [:div.vis-timeline-controls-select-unit
+     [:label
+      {:for "timeline-select-unit"}
+      "Time Unit"]
+     (into [:select
+            {:id "timeline-select-unit"
+             :value current-unit
+             :on-change #(dispatch [:status/set-timeline-unit
+                                    (fns/ps-event-val %)])}]
+           (for [unit ["year"
+                       "month"
+                       "day"
+                       "hour"
+                       "minute"
+                       "second"]]
+             [:option
+              {:value unit
+               :key (str "unit-" unit)}
+              (cs/capitalize unit)]))]))
+
+(defn- pick-datetime
+  [input-id
+   label
+   value
+   on-change
+   min-datetime
+   & [max-datetime]]
+  [:div.vis-timeline-controls-pick-datetime
+   [:label
+    {:for input-id}
+    label]
+   [:input
+    (cond-> {:id input-id
+             :type "datetime-local"
+             :value value
+             :on-change on-change
+             :min min-datetime}
+      max-datetime (assoc :max max-datetime))]])
+
+(defn timeline-pick-since
+  []
+  [pick-datetime
+   "timeline-pick-since"
+   "Since"
+   @(subscribe [:status.params/timeline-since-local])
+   #(dispatch [:status/set-timeline-since
+               (fns/ps-event-val %)])
+   "1970-01-01T00:00:00"
+   @(subscribe [:status.params/timeline-until-local])])
+
+(defn timeline-pick-until
+  []
+  [pick-datetime
+   "timeline-pick-until"
+   "Until"
+   @(subscribe [:status.params/timeline-until-local])
+   #(dispatch [:status/set-timeline-until
+               (fns/ps-event-val %)])
+   @(subscribe [:status.params/timeline-since-local])])
```

Comment:
Huh somehow these limits aren't 100% working:
![Screen Shot 2023-02-21 at 6 04 11 PM](https://user-images.githubusercontent.com/32022576/220478723-5c9ea497-e14e-4b4c-8b7a-f30829cf98fb.png)

Reply:
That looks like some kind of bug in the actual browser implementation. In only seems to stop you if you modify the date, if you modify the time it lets you venture into invalid territory. Weird. I'll put logic in the handler to make sure that can't happen.

Reply:
You can see it being invalid if you inspect the source

    <input id="timeline-pick-since" type="datetime-local" min="1970-01-01T00:00:00" max="2023-02-22T15:03:37" value="2023-02-28T16:59:45">

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/c64d87b242870d818eb5651fce51c86ba05fd964